bundles/org.codechecker.eclipse.plugin.plugin/.project

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.codechecker.eclipse.plugin.plugin</name>
```

Comment:
Do we need the two plugins here in the naming? Can it be done to move the package one level up?

Reply:
Refactored the whole thing, now the project base name is org.codechecker.eclipse.